packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec

```diff
@@ -34,6 +34,13 @@ BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 Provides: foreman-plugin-%{plugin_name} = %{version}
+%if 0%{?fedora} >= 27 || 0%{?rhel} >= 8
```

Comment:
Can you move it below the comment? Within the block it's autogenerated and replaced on package bumping so it'd be dropped on the next package update.

Reply:
like this?

Reply:
Exactly.